.travis.yml

```diff
@@ -1,21 +1,8 @@
 language: node_js
+
 node_js:
-  - "6.0"
-  - "5.0"
-  - "4.0"
-  - "iojs"
-  - "0.12"
-  - "0.11"
-  - "0.10"
-  - "0.8"
-  - "0.6"
+  - "7"
+  - "6"
+  - "4"
 
 sudo: false
-  
-before_install:
-  npm install -g npm@'>=1.4.3'
-
-matrix:
-  allow_failures:
-    - node_js: "0.8"
-    - node_js: "0.6"
```

Comment:
Let's leave `.travis.yml` alone in this PR, especially since I just [updated it](https://github.com/benjamn/ast-types/blob/master/.travis.yml), so there may be rebase conflicts.